static/bidder-info/improvedigital.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "http://ad.360yield.com/pbs"
+endpoint: "http://ad.360yield.com/{PublisherId}pbs"
```

Comment:
Quick question: Why not add the `/` as part of the URL itself rather than appending it in the `buildEndpointURL` method

Reply:
When no publisher ID is defined, URL should be `http://ad.360yield.com/pbs`. So, if I add `/` as part of URL, the URL would be `http://ad.360yield.com//pbs`.